main.go

```diff
@@ -65,10 +64,30 @@ func main() {
 		data, err = decrypt.DataWithFormat(b, format)
 
 		if err != nil {
-			_, _ = fmt.Fprintf(os.Stderr, "failed decrypting file '%s': %s\n", file, err.Error())
-
 			if manifest.FailSilently {
-				failed = true
+				var yamlData map[string]interface{}
+				yErr := yaml.Unmarshal(b, &yamlData)
+				if yErr != nil {
+					_, _ = fmt.Fprintf(os.Stderr, "error unmarshalling manifest content: %q \n%s\n", yErr, b)
+					os.Exit(1)
+				}
+				delete(yamlData, "sops")
+				for k, v := range yamlData {
+					if k == "stringData" || k == "data" {
+						tempList := make(map[string]string)
+						for a, _ := range v.(map[string]interface{}) {
+							tempList[a] = "SECRET"
```

Comment:
> It's a big change and sorry for being not very clear in the beginning, but I think we should keep how fail-silently works now and add this under another flag. A bit later we can remove fail-silently and keep only this, but for less disturbances in argyle-k8s workflows let's do it separately from this PR.
> 
> One other concern I have is setting the flag in secret yamls. `fail-silently` has grep & sed to change the values for `make check`, maybe that's ok for this as well but it feels clunky to me. Weather to generate dummy secrets or just error out on failed decryption depends on the environment where ksops is running, so it almost feels like it should be controled through an environment variable GENERATE_DUMMY_SECRETS and not values in yamls. @nathanmartins @kpocius WDYT?

sure, I will go with something more obvious to ksops. e.g `KSOPS_GENERATE_DUMMY_SECRETS`

Reply:
>In case it's under data the values have to be valid base64, so "SECRET" could trigger a verification error. Maybe just "U0VDUkVU" would work instead. For stringData this is fine.

fixed here https://github.com/argyle-engineering/ksops/pull/1/commits/c6d30701903a3856f43410320ec5cbfd74fce97f